packages/ui/src/components/va-dropdown/hooks/useDropdown.ts

```diff
@@ -75,6 +75,7 @@ export const useDropdown = (
             const availableWidth = reference.getBoundingClientRect().width
             Object.assign(elements.floating.style, {
               maxWidth: `${availableWidth}px`,
+              minWidth: '100%',
```

Comment:
minWidth: 100% for some reason have more importance than maxWidth.

<img width="1215" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/23530004/3a8788c8-8e80-4e64-8868-b0c88e03ac8c">


```suggestion
              width: `${availableWidth}px`,
```